src/components/Library.js

```diff
@@ -1,39 +1,35 @@
-import React, { Component } from 'react';
-import { Link }  from 'react-router-dom';
-import albumData from './../data/albums';
-import '.././styles/library.css';
-import '.././styles/main.css';
+import React, {Component} from "react";
+import {Link} from "react-router-dom";
+import albumData from "./../data/albums";
+import ".././styles/library.css";
+import ".././styles/main.css";
 
 class Library extends Component {
-    constructor(props) {
-        super(props);
-        this.state = { albums: albumData};
-    }
-    render () {
-        return (
-                <main style={{backgroundColor:"red"}} className="collection">
-                
-                <section className="album-covers container clearfix">
-                {
-                this.state.albums.map( (album, index) =>
-                                <div className= "collection-album-container column fourth" key={index}>
-                                    <p>
-                                      <img src={album.albumCover} alt={album.title}/>
-                                        <Link to={`/album/${album.slug}`} >
-                                            <div className="collection-album-info caption"></div>
-                                        </Link>
-                                      <div className="album-name">{album.title}</div>
-                                      <a href="#">{album.artists}</a>
-                                      {album.songs.length} songs
-                                    </p>
-                                </div>
-                                    
-                                      )
-                }
-                </section>
-        </main>
-                );
-    }
+  constructor(props) {
+    super(props);
+    this.state = {albums: albumData};
+  }
+  render() {
+    return (
+      <main style={{backgroundColor: "red"}} className="collection">
+        <section className="album-covers container clearfix">
+          {this.state.albums.map((album, index) => (
+            <div className="collection-album-container column fourth" key={index}>
+              <p>
+                <img src={album.albumCover} alt={album.title} />
+                <Link to={`/album/${album.slug}`}>
+                  <div className="collection-album-info caption" />
```

Comment:
This is breaking everthing. Deleted the `Link` and all divs inside the the `<p>` on line 18